#38: Add extension (#39)
exasol/oracle-virtual-schema: doc/changes/changes_2.4.3.md

## Summary

> This release adds integration tests using Exasol 8 and an extension for the Extension Manager. Please note that the extension expects the Oracle JDBC driver `ojdbc8.jar` in BucketFS. If this file is missing, the extension won't be available in Extension Manager.

Reviewer: ```suggestion
This release adds integration tests using Exasol 8 and the extension code making VSORA available in the [Exasol Extension Manager](https://github.com/exasol/extension-manager). Please note that the VSORA extension expects the Oracle JDBC driver `ojdbc8.jar` in BucketFS. If this file is missing, the extension won't be available in Extension Manager.
```
